code/core/system/proxy-fh16/src/CANMessageDataStore.cpp

```diff
@@ -77,6 +78,10 @@ void CanMessageDataStore::add(odcore::data::Container &container) {
     m_enabled = true;
 
     if (container.getDataType() == opendlv::proxy::ActuationRequest::ID()) {
```

Comment:
Now the write access to the CAN device is granted only if the read-only mode is not engaged.